vars/virtualenv.groovy

```diff
@@ -1,11 +1,21 @@
 import com.ableton.VirtualEnv
 
 
+/**
+ * Create a new {@link com.ableton.VirtualEnv} instance.
+ * @param python Python version or absolute path to Python executable.
+ * @param pyenvRoot Path of the pyenv installation.
+ * @return New {@link com.ableton.VirtualEnv} instance.
+ */
+VirtualEnv create(String python, String pyenvRoot) {
+  return VirtualEnv.create(script, python, pyenvRoot)
```

Comment:
Where is `script` coming from?

Reply:
Good catch @dgl-ableton! It should be:

```suggestion
  return VirtualEnv.create(this, python, pyenvRoot)
```

Reply:
Addressed in https://github.com/AbletonDevTools/python-pipeline-utils/pull/56/commits/5dde137bc6fa4b88dc716ebd3015019290443784.